src/code/client.coffee

```diff
@@ -486,9 +487,12 @@ class CloudFileManagerClient
       @state.currentContent?.copyMetadataTo envelopedContent
       @_ui.downloadDialog @state.metadata?.name, envelopedContent, callback
 
-  getDownloadUrl: (content, includeShareInfo) ->
+  getDownloadUrl: (content, includeShareInfo, mimeType) ->
```

Comment:
Since you test mimeType on line 492 (`if mimeType.indexOf("image") >= 0`) and it is an optional parameter I'd set the default value here:

`, mimeType = 'text/plain') ->`

and get rid of the `mimeType or 'text/plain'` on line 513.